features/gift_exchanges/challenge_yuletide.feature

```diff
@@ -521,6 +533,7 @@ Feature: Collection
 
   # Mod checks for unfulfilled assignments, and gets pinch-hitters to do them.
   When I am logged in as "mod1"
+    And all indexing jobs have been run
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.